Cargo.lock

```diff
@@ -309,23 +300,23 @@ dependencies = [
 
 [[package]]
 name = "gimli"
-version = "0.28.0"
+version = "0.28.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "6fb8d784f27acf97159b40fc4db5ecd8aa23b9ad5ef69cdd136d3bc80665f0c0"
+checksum = "4271d37baee1b8c7e4b708028c57d816cf9d2434acb33a549475f78c181f6253"
 
 [[package]]
 name = "h2"
-version = "0.3.21"
+version = "0.3.24"
```

Comment:
Listing it as a dependency in `nss/Cargo.toml` was not preventing cargo from pulling this other one, so I just ran `cargo update`, which updates the dependencies in the Lockfile to their latest SemVer version and I think we are good to go.